oidc/client.go

```diff
@@ -66,11 +66,16 @@ type ClientConfig struct {
 }
 
 func NewClient(cfg ClientConfig) (*Client, error) {
+	ru, err := phttp.ParseNonEmptyURL(cfg.RedirectURL)
```

Comment:
@sym3tri Later for the party, but why redirectURL is required here? E.g. I think I can use the client for just token verification or token refreshing, which doesn't need the redirectURL info.